### DISCUSSION TITLE
Two context menu appears everty time I execute a query with "Control + Enter" combination keys, on MacOS Sequoia 15.0, Spanish.

### DISCUSSION THREAD

**Discussion by SalvadorAlonsoOrtega** [dbeaver/dbeaver#35687] [Q&A] [answered]
Hi, with version Versión24.2.1.202409221725 on MacOS Sequoia 15.0, I notice a new behavior when executing querys with shortcuts. 
If you execute Control + Enter combination keys, the query will be executed but the context menu appears twice. 

First this menu:
![Captura de pantalla 2024-09-23 a la(s) 9 14 32 a m](https://github.com/user-attachments/assets/ea4edf79-9fcd-4648-a997-79f9ef20fa2b)

and this later:
![Captura de pantalla 2024-09-23 a la(s) 9 14 39 a m](https://github.com/user-attachments/assets/332f7f59-8c40-4c7f-8cfa-9328b62b029c)

If you run the query with toolbar, no problem.

Is there a way to avoid this behavior?

Thanks in advance.

**Comment by raaaaaw** [comment, marked as answer]
Duplicate already resolved here -> https://github.com/dbeaver/dbeaver/issues/35109

**Reply by SalvadorAlonsoOrtega** [reply]
Thank you so much @raaaaaw . 
Best regard.
